tox.ini

```diff
@@ -24,30 +27,45 @@ python =
     3.8: py38
     3.9: py39
 
-[testenv:py37-bandit]
+[gh-actions:env]
+DJANGO =
+    2.2: django22
+    3.0: django30
+    3.1: django31
+
+[testenv:bandit]
 description = PyCQA security linter
 deps = bandit<1.6
 commands = - bandit -r --ini tox.ini
 ignore_errors = true
 
-[testenv:py37-docs]
+[testenv:docs]
 description = Build the HTML documentation
 deps = sphinx
 commands = sphinx-build -b html -d docs/_build/doctrees docs docs/_build/html
 whitelist_externals = make
 
-[testenv:py37-flake8]
+[testenv:flake8]
 description = Static code analysis and code style
 deps = flake8
 commands = flake8
 
-[testenv:py37-readme]
+[testenv:readme]
 description = Ensure README renders on PyPI
 deps = twine
 commands =
     {envpython} setup.py -q sdist bdist_wheel
     twine check dist/*
 
+[testenv:clean]
+description = Clean up bytecode and build artifacts
+deps = pyclean
+commands =
+    pyclean -v {toxinidir}
+    rm -rf .tox/ build/ dist/ django_analytical.egg-info/ .coverage coverage.xml
+whitelist_externals =
+    rm
```

Comment:
Is this clean command just for development purposes?

Reply:
Yes, this is only for local use.